change/@fluentui-react-datepicker-compat-ea635309-a722-4860-bdc8-54b8de89c9c7.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 15 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27731&commitId=97e2373a219f2c1f21dbeeeeb32eb888c485b124&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1918 screenshots unchanged, 0 screenshots with different dimensions and 15 screenshots with visible difference.


  <details><summary><b>unknown</b> 15 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|DatePicker Compat.DateRange- day.default.chromium.png|189| Changed|
|DatePicker Compat.DateRange- month - Dark Mode.default.chromium.png|140| Changed|
|DatePicker Compat.DateRange- month - High Contrast.default.chromium.png|119| Changed|
|DatePicker Compat.DateRange- month.default.chromium.png|140| Changed|
|DatePicker Compat.DateRange- week.default.chromium.png|164| Changed|
|DatePicker Compat.DateRange- work week.default.chromium.png|164| Changed|
|DatePicker Compat.default - Dark Mode.opened.chromium.png|119| Changed|
|DatePicker Compat.default - High Contrast.opened.chromium.png|143| Changed|
|DatePicker Compat.default.opened.chromium.png|164| Changed|
|DatePicker Compat.marked dates - Dark Mode.default.chromium.png|200| Changed|
|DatePicker Compat.marked dates - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.marked dates.default.chromium.png|189| Changed|
|DatePicker Compat.showWeekNumbers - Dark Mode.default.chromium.png|200| Changed|
|DatePicker Compat.showWeekNumbers - High Contrast.default.chromium.png|197| Changed|
|DatePicker Compat.showWeekNumbers.default.chromium.png|189| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>

Reply:
Changes are expected, they are just focus indicators showing up.